cmd/helm/init.go

```diff
@@ -36,7 +36,10 @@ import (
 )
 
 const initDesc = `
-This command sets up local configuration in $HELM_HOME (default ~/.helm/).
+This command sets up local configuration in $XDG_CONFIG_HOME/helm
+($HOME/.config/helm on Linux, %APPDATA%\helm on Windows and
+$HOME/Library/Preferences on OSX).
+NOTE: if there is old-style "$HOME/.helm" directory, it will be preffered.
```

Comment:
Would you mind changing this to:

> NOTE: if $HOME/.helm exists, it will be used instead of $XDG_CONFIG_HOME/helm.

Reply:
Also `~/.helm` not be used during `helm init`.  Init should always create $HELM_HOME || $XDG_CONFIG_HOME/helm.  `~/.helm` should only be used as a fallback for lookup.